### DISCUSSION TITLE
Why am I getting a core error

### DISCUSSION THREAD

**Discussion by kaushikc44** [wevm/wagmi#2650] [General]
{context: 'core'} {context: 'core/relayer'}context: "core/relayer"[[Prototype]]: Object Error: WebSocket connection closed abnormally with code: 3000 (Authorization error: Unauthorized: invalid key) this si the error I am getting now sure why ?

**Comment by DorianDeSimone** [comment]
Hi, i'm having the same message, do you find an answer ?
